client/src/components/molecules/EmailAndPassInput/EmailAndPassInput.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Form } from 'react-bootstrap';
+
+const EmailAndPassInput = () => {
+  return (
+    <>
```

Comment:
why fragment?

Reply:
The form would not render without the fragment